src/hotspot/share/classfile/systemDictionary.hpp

```diff
@@ -240,21 +235,12 @@ class SystemDictionary : AllStatic {
                                               TRAPS);
 
   // compute java_mirror (java.lang.Class instance) for a type ("I", "[[B", "LFoo;", etc.)
-  // Either the accessing_klass or the CL/PD can be non-null, but not both.
+  // Either the accessing_klass or the CL can be non-null, but not both.
+  // callee will fill in CL from AK, if they are needed
```

Comment:
```suggestion
  // Callee will fill in CL from accessing_klass, if they are needed.
```

Reply:
fixed. All these comments could use capitalization, but I won't do that here.